src/bin/vip-wp.js

```diff
@@ -114,28 +113,6 @@ const getTokenForCommand = async ( appId, envId, command ) => {
 		} );
 };
 
-const cancelCommand = async ( guid ) => {
```

Comment:
Should we keep this for when we want to add cancel support?

Reply:
As far as I could tell that function is not used anywhere 🤷‍♂️  but I could be missing some context?

Reply:
Sorry, I meant more that if we want to add a `vip wp-cancel` command, this would be useful.

Reply:
I see what you mean.  I can revert it 👍